libs/application/templates/driving-license-duplicate/src/lib/drivingLicenseDuplicateTemplate.ts

```diff
@@ -18,7 +18,7 @@ import { Events, States, Roles } from './constants'
 import { dataSchema } from './dataSchema'
 import { m } from './messages'
 import { ApiActions } from './constants'
-import { FeatureFlagClient, Features } from '@island.is/feature-flags'
+import { FeatureFlagClient } from '@island.is/feature-flags'
 import {
   DrivingLicenseDuplicateFeatureFlags,
   getApplicationFeatureFlags,
```

Comment:
What about this?

And also L#76-87?

Reply:
these are being used for testing on dev with fake data